v2/kafka-to-bigquery/src/test/java/com/google/cloud/teleport/v2/templates/KafkaToBigQueryFlexIT.java

```diff
@@ -0,0 +1,234 @@
+/*
```

Comment:
Can we rename the file to `KafkaToBigQueryFlexJsonIT.java`?

Reply:
Done